### PR TITLE
SONARGO-82 Add "artifactory-configuration" to common Gradle modules

### PR DESCRIPTION
**Pull request by petertrr**
[SONARGO-82](https://sonarsource.atlassian.net/browse/SONARGO-82)

Verified: No significant differences in buildinfo.json and uploaded `.pom` files

[SONARGO-82]: https://sonarsource.atlassian.net/browse/SONARGO-82?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -87,6 +87,7 @@ build_task:
     SONAR_TOKEN: VAULT[development/kv/data/next data.token]
     SONAR_HOST_URL: VAULT[development/kv/data/next data.url]
     DEPLOY_PULL_REQUEST: "true"
+    FORCE_SIGNING: "true"
```
Comment (mstachniuk) [inline .cirrus.yml]: Why is it needed? We don't use it in other projects

File: build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+ * You should have received a copy of the Sonar Source-Available License
+ * along with this program; if not, see https://sonarsource.com/license/ssal/
+ */
+plugins {
+    id("org.sonarsource.cloud-native.code-style-conventions")
+    id("org.sonarsource.cloud-native.artifactory-configuration")
+    id("org.sonarqube") version "6.0.1.5171"
+}
+
+artifactoryConfiguration {
```
Comment (mstachniuk) [inline build.gradle.kts:23]: This is also configured in sonar-go-plugin/build.gradle.kts. Can it be in one place only?

**Review by mstachniuk — APPROVED** [review]
LGTM and see my comments

File: .cirrus.yml (hunk 1 above)
Comment (petertrr) [reply .cirrus.yml]: Forgot to remove after testing, thanks

File: build.gradle.kts (hunk 2 above)
Comment (petertrr) [reply build.gradle.kts:23]: IMO it's better to configure publication in the same project, that is published. However, the Jfrog plugin needs certain configuration on the root project specifically. I don't think it should be all in one place.

Comment (petertrr) [reply build.gradle.kts:23]: Split `artifactory-configuration` into `artifactory-configuration` (applied to the root project only) and `publishing-configuration` (applied to the plugin)

**Pull request merged**